components/nav-link/nav-link.jsx

```diff
@@ -1,18 +1,31 @@
 import React from 'react';
 import { Link } from 'react-router';
 import ReactGA from 'react-ga';
+import PropTypes from 'prop-types';
+
+class NavLink extends React.Component {
+  constructor() {
+    super();
```

Comment:
A few online articles mention if we aren't using `props` in the constructor there's no need to pass `props` to `super()`